### PR TITLE
added energy topic

### PR DESCRIPTION
**Pull request by AFoletti**

### CODE REVIEW

**Comment by AFoletti** [comment]
@oterral please review and merge if OK

**Comment by AFoletti** [comment]
jenkins test

File: src/locales/de.json [hunk 1]
```diff
@@ -120,6 +120,9 @@
 "emapis_service_link_href": "mailto:info@blw.admin.ch",
 "emapis_service_link_label": "info@blw.admin.ch",
 "embed_map": "Karte einbetten",
+"energie": "Energie",
+"energie_service_link_href": "http://www.bfe.admin.ch/geoinformation",
+"energie_service_link_label": "www.map.energie.admin.ch",
```
Comment (oterral) [inline src/locales/de.json]: `lang=de`parameter ?

File: src/locales/rm.json [hunk 2]
```diff
@@ -120,6 +120,9 @@
 "emapis_service_link_href": "mailto:info@blw.admin.ch",
 "emapis_service_link_label": "info@blw.admin.ch",
 "embed_map": "Integrar charta",
+"energie": "",
+"energie_service_link_href": "",
+"energie_service_link_label": "",
```
Comment (oterral) [inline src/locales/rm.json]: No translations? not even german ?

File: src/locales/en.json [hunk 3]
```diff
@@ -120,6 +120,9 @@
 "emapis_service_link_href": "mailto:info@blw.admin.ch",
 "emapis_service_link_label": "info@blw.admin.ch",
 "embed_map": "Embed map",
+"energie": "Energy",
+"energie_service_link_href": "http://www.bfe.admin.ch/geoinformation/index.html?lang=en",
+"energie_service_link_label": "www.map.energy.admin.ch",
```
Comment (oterral) [inline src/locales/en.json]: lang parameter ?

File: src/locales/fr.json [hunk 4]
```diff
@@ -120,6 +120,9 @@
 "emapis_service_link_href": "mailto:info@blw.admin.ch",
 "emapis_service_link_label": "info@blw.admin.ch",
 "embed_map": "Intégrer la carte",
+"energie": "Énergie",
+"energie_service_link_href": "http://www.bfe.admin.ch/geoinformation/index.html?lang=fr",
+"energie_service_link_label": "www.map.energie.admin.ch",
```
Comment (oterral) [inline src/locales/fr.json]: lang parameter ?

File: src/locales/en.json (hunk 3 above)
Comment (oterral) [inline src/locales/en.json]: Forget this one.

**Comment by oterral** [comment]
jenkins test

File: src/locales/rm.json (hunk 2 above)
Comment (AFoletti) [inline src/locales/rm.json]: topic not available in RM

**Comment by AFoletti** [comment]
I added the lang parameter to DE too

**Pull request merged**